src/webgl/p5.Matrix.js

```diff
@@ -458,42 +477,32 @@ p5.Matrix.prototype.mult = function(multMatrix) {
  * @param  {p5.Vector|Float32Array|Number[]} s vector to scale by
  * @chainable
  */
-p5.Matrix.prototype.scale = function() {
-  var x, y, z;
-  var args = new Array(arguments.length);
-  for (var i = 0; i < args.length; ++i) {
-    args[i] = arguments[i];
-  }
+p5.Matrix.prototype.scale = function(x, y, z) {
   //if our 1st arg is a type p5.Vector
```

Comment:
this part reads a little funny (x = x[0] or x = x.x), but I get what you're doing here. I think it's worth a quick note inline that this is done for optimization purposes, so someone doesn't try to clean this up later.